components/sections/ContactUs.jsx

```diff
@@ -0,0 +1,17 @@
+import { LargeButton } from "../ui/Button";
+import { H3 } from "../ui/Headings";
+
+const ContactUs = () => {
+    return (
+        <section className="bg-primary-800 mx-0.5 rounded-8xl gap-16 xl:px-16 px-5 py-8 flex flex-col items-center">
+            <div className="xl:w-239">
+                <H3 text="Ready to talk about your project?" styles=" text-center text-white" />
```

Comment:
You should remove the unnecessary white space when passing `styles` prop

Reply:
Done